tests/test_headers.py

```diff
@@ -472,7 +472,9 @@ def test_response_set_and_get_header(self, client):
             assert result.headers['X-Auth-Token'] == 'toomanysecrets'
 
             assert resource.resp.location is None
-            assert resource.resp.get_header('not-real') is None
+            assert resource.resp.get_header('X-Header-Not-Set') is None
+            assert resource.resp.get_header('X-Header-Not-Set', 'Yes') == 'Yes'
+            assert resource.resp.get_header('X-Header-Not-Set', default='') == ''
 
```

Comment:
Could we also add a test case where the default is provided, but the header *is* set?

Reply:
I went ahead and took care of this.